nerfstudio/process_data/images_to_nerfstudio_dataset.py

```diff
@@ -47,20 +45,24 @@ def main(self) -> None:
 
         # Generate planar projections if equirectangular
         if self.camera_type == "equirectangular":
-            pers_size = equirect_utils.compute_resolution_from_equirect(self.data, self.images_per_equirect)
+            pers_size = equirect_utils.compute_resolution_from_equirect(self.data[0], self.images_per_equirect)
             CONSOLE.log(f"Generating {self.images_per_equirect} {pers_size} sized images per equirectangular image")
-            self.data = equirect_utils.generate_planar_projections_from_equirectangular(
-                self.data, pers_size, self.images_per_equirect, crop_factor=self.crop_factor
+            self.data = (
+                equirect_utils.generate_planar_projections_from_equirectangular(
+                    self.data[0], pers_size, self.images_per_equirect, crop_factor=self.crop_factor
+                ),
             )
             self.camera_type = "perspective"
 
+        assert len(self.data) == 1, "Only one data source is supported for images"
```

Comment:
Should we add support for this in this PR too?

Reply:
Maybe we can split into a seperate PR?